docs/topics_5/maven-prerequisites.adoc

```diff
@@ -9,7 +9,6 @@
 * 8 GB RAM
 * Maven 3.2.5 or later
 
-{MavenName} on macOS has the following additional prerequisites:
+{MavenName} on macOS has the following additional prerequisite:
 
```

Comment:
If there is only one prerequisite, I would turn this into a single sentence instead of a sentence and one bullet point. You don't have to do anything about it. I'll fix it when I turn these into a single module.